packages/payment/src/loaders/providers.ts

```diff
@@ -33,6 +35,12 @@ export default async ({
   const pluginProviders =
     options?.providers?.filter((provider) => provider.resolve) || []
 
+  // Local providers
+  for (const provider of Object.values(providers)) {
+    // TODO: pass options
+    await registrationFn(provider, container, {})
+  }
```

Comment:
thought: If the only local provider is `system`, I think we could leave this out and just inject the service directly into the payment provider. Similar to Google and EmailPass in the auth module

Reply:
could you elaborate please, I think we are doing similar thing here as with the auth module